src/modules/git_commit.rs

```diff
@@ -335,4 +339,76 @@ mod tests {
         assert_eq!(expected, actual);
         Ok(())
     }
+
+    #[test]
+    fn test_latest_tag_shown_with_tag_enabled() -> io::Result<()> {
+        use std::{thread, time};
+
+        let repo_dir = fixture_repo(FixtureProvider::GIT)?;
+
+        let mut git_commit = Command::new("git")
+            .args(&["rev-parse", "HEAD"])
+            .current_dir(&repo_dir.path())
+            .output()?
+            .stdout;
+        git_commit.truncate(7);
+        let commit_output = str::from_utf8(&git_commit).unwrap().trim();
+
+        Command::new("git")
+            .args(&["tag", "v2", "-m", "Testing tags v2"])
+            .current_dir(&repo_dir.path())
+            .output()?;
+
+        // Wait one second between tags
+        thread::sleep(time::Duration::from_millis(1000));
```

Comment:
I'm a little bit suspicious of this line because it seems to rely on wallclock properties (e.g. if a daylight-savings or multiple leap-second timechange occur while the sleep is happening, does the test still work?) but I don't see a better way of testing it at the moment since you can't artifically specify creation times in git.

Nothing to be done for the moment, just something to remember if we see this test start failing in GH Actions for no apparent reason.

Reply:
Totally agree with you! It's a little bit _dirty_ solution for generate two tags in a row and then make sure starship is showing the latest one, but I didn't found another way of testing it...

I hope this _trick_ won't broke test, but honestly, I don't know how GitHub Actions will react in those all scenarios... 😓

Reply:
Yeah, I don't think we really have a choice but to hope that this keeps working (it definitely works for now, since the GH Actions build is passing). Just making a note here in case it breaks in a future release so that we have an easier time tracking it down.